src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -639,6 +626,63 @@ private BoundExpression BindExpressionInternal(ExpressionSyntax node, Diagnostic
             }
         }
 
+        private BoundExpression BindTupleExpression(TupleExpressionSyntax node, DiagnosticBag diagnostics)
+        {
+            var arguments = node.Arguments;
+            if (arguments.Count < 2)
+            {
+                // this should be a parse error already.
+                return BadExpression(node);
```

Comment:
We should bind arguments anyway and capture result a s children of the BadExpression, this is needed for SemanticModel, IOperation, etc.